configure.sh

```diff
@@ -29,7 +29,8 @@ The following flags enable optional features (disable with --no-<option name>).
   --optimized              optimize the build
   --debug-symbols          include debug symbols
   --valgrind               Valgrind instrumentation
-  --debug-context-mm       use the debug context memory manager
+  --debug-context-memory-manager
```

Comment:
This option is named debug-context-mm on purpose since the old name (debug-context-memory-manager) is way to long. I don't want this to be renamed.

Reply:
Oh, I see, I forgot to rename the other occurrences. I'd suggest to maybe leave the name as is and rename the other occurrences?

Reply:
Sure, I am fine with that :) I just noticed that the help didn't match the actual name when I tried to use it.